vmdb/spec/models/ems_refresh/refreshers/openstack_infra_refresher_rhos_juno_spec.rb

```diff
@@ -48,15 +49,16 @@ def assert_table_counts
     Lan.count.should                 == 0
     MiqScsiLun.count.should          == 0
     MiqScsiTarget.count.should       == 0
-    Network.count.should             == 0
+#    Network.count.should             == 1
```

Comment:
I added test data for networks and flavors as part of this refactor so that other developers don't have to repeat this process. The collection is not functional until additional modeling changes are in place, so that is why they are commented out.